solr/packaging/build.gradle

```diff
@@ -86,12 +86,20 @@ distributions {
         include "NOTICE.txt"
       })
 
+      from(project(":solr:contrib").projectDir, {
```

Comment:
the `modules` directory should have a build.gradle that creates the `local_packages.json` in it's build directory, then puts that into a gradle configuration. Then this file can refer to that configuration.

same with `local_packages.json` and `manifest.json` below.